src/components/previewers/ImagePreviewer.jsx

```
@@ -0,0 +1,150 @@
+            </div>
+          </div>
+        </div>
+
+        {/* Action Buttons */}
+        {!isTop && (
+          <div className="action-buttons-container mt-3">
+            <button
+              onClick={handleLike}
+              className="action-button"
+              aria-label="Like"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)